src/components/footer/FooterOne.jsx

```diff
@@ -0,0 +1,318 @@
+/**
+* Footer of the Myntra Website
+* This is the first footer component
+* @returns the first half of the footer component to FooterMain function
+* @author Sanjana Rao
+*/
+import React from 'react';
+import './footerOne.css';
+import TwitterIcon from '@mui/icons-material/Twitter';
+import FacebookIcon from '@mui/icons-material/Facebook';
+import YouTubeIcon from '@mui/icons-material/YouTube';
+import InstagramIcon from '@mui/icons-material/Instagram';
+import mobile from '../../assets/mobile.png';
+import returnOrder from '../../assets/return.png';
+import original from '../../assets/original.png';
+
+export default function FooterOne() {
+  const year = new Date().getFullYear();
+  return (
+    <div className="footer">
+      <div className="footerFirst">
+        <div className="footerFirstOne">
+          <h5 className="footerHeading">Online Shopping</h5>
+          <li className="footerItems">
+            <a href="/men">Men</a>
+          </li>
+          <li className="footerItems">
+            <a href="/women">Women</a>
+          </li>
+          <li className="footerItems">
+            <a href="/kids">Kids</a>
+          </li>
+          <li className="footerItems">
+            <a href="/home">Home & Living</a>
+          </li>
+          <li className="footerItems">
```

Comment:
Each division has <li><a></a></li> , Can you loop through an array data to get this done instead of writing the same code again and again.